.github/workflows/main.yml

```diff
@@ -30,7 +30,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -v --editable .[lint]
+          pip install -v --editable '.[lint, test]'
```

Comment:
```suggestion
          pip install -v --editable .[lint,test]
```

Reply:
I guess the whitespace is the issue

Reply:
I checked again and the syntax causes problems in some shells, e.g. zsh. where `pip install -v --editable .[lint,test]` also doesn't work without the whitespace. I think I don't like whitespace dependent shell commands :-) So, I would just leave the apostrophes, but I can get rid of the whitespace.